cmd/crc/cmd/daemon.go

```diff
@@ -217,6 +218,10 @@ func run(configuration *types.Configuration) error {
 			if err != nil && !errors.Is(err, net.ErrClosed) {
 				logging.Errorf("unixgramListener error: %v", err)
 			}
+			if errors.Is(err, drivers.ErrNotImplemented) {
+				cancel()
```

Comment:
`unixgramListener` did nothing, so calling `cancel` is likely to be a noop

Reply:
it is a noop. Not adding it was resulting in `govet` errors.